examples/asr/transcribe_speech.py

```diff
@@ -213,6 +213,17 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     compute_timestamps = cfg.compute_timestamps
     compute_langs = cfg.compute_langs
 
+    # Check whether model and decoder type match
+    if isinstance(asr_model, EncDecCTCModel):
```

Comment:
If the model is CTC, then decoder_type can be just ignored as there is no way to use any other decoder. So maybe we don't need these checks?

You may just check the value in the decoder_type to make sure it is one of the values of "ctc" or "rnnt". This check can happen for all models, no need to check the type of the model.

Reply:
I actually don't want the user to pass CTC model + rnnt and it successfully run by ignoring the decoder_type